client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -36,6 +39,48 @@ export default React.createClass( {
 		this._pressableStore = pressableStoreStep;
```

Comment:
@dmsnell  I don't see any reason for using leading underscore here, but I might be missing something since I didn't add that? Here is the original PR in which it was introduced: https://github.com/Automattic/wp-calypso/pull/6340